features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/utils/DrawableUtilsTest.kt

```diff
@@ -379,14 +373,19 @@ internal class DrawableUtilsTest {
     }
 
     @Test
-    fun `M not use original drawable W createBitmapOfApproxSizeFromDrawable`() {
+    fun `M uses original drawable W createBitmapOfApproxSizeFromDrawable`() {
+        /**
```

Comment:
this is to be removed before the merge? it is better to add it as PR comment rather than code comment.